src/pages/loading-waiting-states/spinner-overlays/eulaText.tsx

```diff
@@ -0,0 +1,5 @@
+export const eulaDetails = {
+    title: 'End User License Agreement',
+    loadingMessage: 'Loading EULA...',
```

Comment:
Oh I didn't see this part yesterday. We can probably have these hardcoded in the other file as well and then this file can just be:
```
export const eulaText = 'THIS EULA.....'
```